sagemaker-pipelines/data-ingestion/feature-store-data-ingestion-via-sagemaker-model-building-pipeline.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<ul><li>Consider adding a "Key Concepts" section here that defines terms like "Feature Group", "Pipeline", etc...</li><li>DataWrangler is used but not mentioned here.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>